android/app/src/main/java/org/fossasia/openevent/common/utils/Utils.kt

```diff
@@ -85,15 +85,15 @@ object Utils {
             if (isBaseUrlEmpty) {
                 swipeRefreshLayout.isEnabled = false
             } else {
-                StrategyRegistry.instance.eventBusStrategy!!.eventBus.register(`object`)
+                StrategyRegistry.instance.eventBusStrategy?.getEventBus()?.register(`object`)
```

Comment:
Why not ?.eventBus

Reply:
Because I could not set a default getter implementation because the getter referenced the eventBus and that turned into infinite recursion. So I had to create an explicit getEventBus method

Reply:
Why would it recur? That doesn't make sense

Reply:
For some reason, accessing the property from inside the getter calls the getter again.

Reply:
Yeah, android studio showed the error. But I found a solution using the 'field' naming.

Reply:
Why are you using that?

Reply:
Becuase we need to do this
if (field == null) {
                field = Bus()
            }
            return field

Reply:
use lazy delegator

Reply:
We don't need the lazy approach here. We can just use the field attribute.